tests/configfiles/test_files/default_rule.yaml

```diff
@@ -0,0 +1,225 @@
+---
```

Comment:
could you call the directory `dockerfile_lint_test_files` or similar that somehow incorporates the tool name / bear name? Other bears might want to add tests in the future too, having specific directories makes it easier to organize test files.